main.go

```diff
@@ -221,7 +226,7 @@ func runGRPCServer(ctx context.Context, deps *gRPCServerDeps) {
 	dbaasv1beta1.RegisterLogsAPIServer(gRPCServer, managementdbaas.NewLogsService(deps.db, deps.dbaasClient))
 	dbaasv1beta1.RegisterComponentsServer(gRPCServer, managementdbaas.NewComponentsService(deps.db, deps.dbaasClient, deps.versionServiceClient))
 
-	platformService, err := platform.New(deps.db, deps.supervisord, deps.grafanaClient)
+	platformService, err := platform.New(deps.db, deps.supervisord, deps.grafanaClient, cfg)
```

Comment:
let's create this service in `main()`, so we have consistency in service creation.

Reply:
in this case we don't have to pass `cfg`

Reply:
I don't want to make this change now, this method is blocking goroutine will be blocked till a shutdown.
and this service needs gRPCServer as a dependency

Reply:
then let's make `cfg` part of `deps`

Reply:
ok